.github/workflows/website-validation.yml

```diff
@@ -63,6 +63,31 @@ jobs:
           pnpm i --frozen-lockfile
           pnpm lint
 
+  test:
+    runs-on: ubuntu-latest
```

Comment:
Would love a comment on what this step does / why you need an extra test step, isnt npm run test going to executed by our build step?

Reply:
I left some job-by-job comments. The build job doesn't run the tests; it just builds the site. Subsequent jobs that depend on the build download the artifact it publishes.